packages/graphile-build-pg/src/QueryBuilder.js

```diff
@@ -798,7 +804,11 @@ order by (row_number() over (partition by 1)) desc`;
     } else if (type === "from") {
       if (this.data.from) {
         const f = this.data.from;
-        this.compiledData.from = [callIfNecessary(f[0], context), f[1]];
+        let _f = [callIfNecessary(f[0], context), f[1]];
+        if (this.ignoreSchemas) {
+          _f[0] = { ..._f[0], names: _f[0].names.slice(-1) };
+        }
+        this.compiledData.from = _f;
```

Comment:
This assumes the from is an identifier; sometimes it’s a subquery or function call or similar. I don’t think this approach will work in general.